src/include/distributed/coordinator_protocol.h

```diff
@@ -117,6 +117,13 @@ typedef enum IncludeSequenceDefaults
 } IncludeSequenceDefaults;
 
 
+typedef enum IncludeIdentityDefaults
+{
+	NO_IDENTITY = 0,
+	INCLUDE_IDENTITY_AS_SEQUENCE_DEFAULTS = 1,
+	INCLUDE_IDENTITY = 2
+} IncludeIdentityDefaults;
```

Comment:
I am not sure about the name `IncludeIdentityDefaults`. After all, these are not defaults because `atthasdef` is false for columns that are generated as identities. How about something like `IncludeGeneratedAsIdentity`? Or even simply `IncludeIdentity`?